### PR TITLE
Include `golang` in the test image

### PR DESCRIPTION
**Pull request by chendave**
This is important for a project written by `golang`.

see the discussion here: https://github.com/kubernetes/kubernetes/pull/88399#discussion_r383107339

### CODE REVIEW

**Comment by chendave** [comment]
cc @odinuge

File: images/bootstrap/Dockerfile [hunk 1]
```diff
@@ -48,6 +48,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     xz-utils \
     zip \
     zlib1g-dev \
+    golang \
```
[the remark this answers is not in the document]
Comment (chendave) [reply images/bootstrap/Dockerfile]: done! and thanks for the comment.

File: images/bootstrap/Dockerfile [hunk 2]
```diff
@@ -33,6 +33,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     curl \
     file \
     git \
+    golang \
```
Comment (BenTheElder) [inline images/bootstrap/Dockerfile]: this should not have merged.
go is already installed in kubekins, and we use the correct version for the kubernetes branch, NOT whatever is in debian (which is probably super old)

**Pull request merged**